### PR TITLE
Test guidepup select

### PR DESCRIPTION
**Pull request by bruno-sch**
## Proposed changes

Test guidepup select

## Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Refactoring (fix on existing components or architectural decisions)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Documentation Update (if none of the other choices apply)

### CODE REVIEW

File: showcases/screen-reader/__snapshots__/macos/webkit/DBSelect-default-1.txt
Comment (bruno-sch) [inline showcases/screen-reader/__snapshots__/macos/webkit/DBSelect-default-1.txt:1]: @mfranzke and @nmerget 

After opening the select, the hidden option (1/3) is highlighted and mentioned first. This is an option with the attribute `hidden` and without text and `value` attribute. Without voiceover, it is not visible or selectable. See screenshots attached. Does that make sense?

Unfolded outpiut:
```
"Label Functional pop up button",
"menu 3 items ✓. To choose this menu item, press Control-Option-Space",
"Option 1. To choose this menu item, press Control-Option-Space",
"Option 2. To choose this menu item, press Control-Option-Space",
"To choose this menu item, press Control-Option-Space. Closing menu Option 2 Label Functional pop up button"
```

Screenshot 2024-08-28 at 11 41 03
Screenshot 2024-08-28 at 11 41 30

Comment (nmerget) [reply showcases/screen-reader/__snapshots__/macos/webkit/DBSelect-default-1.txt:1]: We use this empty option to enable the floating label.
This is a problem we should find a solution for this...

**Review by nmerget — DISMISSED** [review]

File: showcases/screen-reader/__snapshots__/macos/webkit/DBSelect-default-1.txt
Comment (mfranzke) [reply showcases/screen-reader/__snapshots__/macos/webkit/DBSelect-default-1.txt:1]: @nmerget what's the solution for this?

Comment (nmerget) [reply showcases/screen-reader/__snapshots__/macos/webkit/DBSelect-default-1.txt:1]: No, but we open up a new story for this. It seems to be a bigger issue

Comment (mfranzke) [reply showcases/screen-reader/__snapshots__/macos/webkit/DBSelect-default-1.txt:1]: https://github.com/db-ui/mono/issues/3116

**Review by nmerget — APPROVED** [review]

**Pull request merged**